src/uEMEP_subgrid_EMEP.f90

```diff
@@ -1894,8 +1894,8 @@ subroutine uEMEP_subgrid_EMEP_from_in_region
                 ii_frac_target = (x_temp-var1d_nc(ii,x_dim_nc_index))/dgrid_nc(x_dim_nc_index)
                 jj_frac_target = (y_temp-var1d_nc(jj,y_dim_nc_index))/dgrid_nc(y_dim_nc_index)
                 ! verify that this is within 0-1 (if not, crossreference has gone wrong...)
-                if (ii_frac_target < -0.5 .or. ii_frac_target > 0.5 .or. jj_frac_target < -0.5 .or. jj_frac_target > 0.5) then
-                    write(unit_logfile,'(A,2I12)') 'Something went wrong with locating target subgrid within EMEP grid!',ii_frac_target,jj_frac_target
+                if (ii_frac_target < -0.51 .or. ii_frac_target > 0.51 .or. jj_frac_target < -0.51 .or. jj_frac_target > 0.51) then
```

Comment:
How big is the margin? Here you could also use abs(ii_frac_target - 0.5) > epsilon0. Or some other tolerance value if 1e-6 is too restrictive?

Reply:
I agree that we can reduce the number of checks by using "abs". However, <epsilon0 is too strict: In the RI Urbans run, I found at least one case where the deviation outside the range [-0.5, 0.5] was bigger than epsilon0 (I think it was in the order of 1e-4). This check really is meant just to verify that we have found the closest EMEP grid to the target subgrid. If we are at 0.51, it just means that the next EMEP grid is a little bit closer, but in practice the subgrid is almost exactly in the middle of the two grids.

Reply:
Ok, I guess this is OK then. We should consider defining a number of tolerance levels in uemep_constants